__tests__/unittest/__snapshots__/templates.test.js.snap

```diff
@@ -42,9 +42,9 @@ exports[`templates fCreateAPIResourceTemplate should create API resource templat
     "extensible": {
       "supported": "no",
     },
-    "ordId": "customer:apiResource:undefined.MyService:v1",
+    "ordId": "customer:apiResource:MyService:v1",
```

Comment:
Why have we suddenly `customer` as a namespace here? This is just a vendor namespace, there needs to be a full system namespace at least, so `customer.something`. Here you would use the namespace from CDS RC

Reply:
Is this because in this test, we have no cdsrc and use full defaults? In that case we should default to something like `customer.<packageNameEscaped>`

Reply:
@Fannon @zongqichen :Renamed into `customer.testNamespace` since it was used in one of the assertions from above. Actually, the string `customer.testNamespace-123` was used, but it doesn't change the thing.